app/experimenter/experiments/migrations/0034_auto_20190319_2122.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 2.1.7 on 2019-03-19 21:22
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("experiments", "0033_auto_20190319_1820")]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="experiment", name="dashboard_image_url"
+        ),
+        migrations.RemoveField(model_name="experiment", name="dashboard_url"),
+        migrations.RemoveField(
+            model_name="experiment", name="enrollment_dashboard_url"
+        ),
+        migrations.RemoveField(model_name="experiment", name="total_users"),
+        migrations.AlterField(
```

Comment:
I can't understand this change. Is it a bad rebase or something.
The `client_matching` isn't touched in the diff of `models.py` as far as I can see. 
Its text comes from `CLIENT_MATCHING_DEFAULT` but that string hasn't changed either. 

For what it's worth, I'm reasonably confident you can just delete this 5th element on this `operations` list.

Reply:
Yes this was missed in an earlier PR and was fixed in yet another PR so when I rebase this will go away.  I've updated CI to block on missed migrations so this doesn't happen in the future.